cli/app.go

```diff
@@ -1909,8 +1909,9 @@ var app = &cli.App{
 							Usage: "filter logs by levels (e.g., info, warn, error)",
 						},
 						&cli.StringFlag{
-							Name:  generalFlagStart,
-							Usage: "ISO-8601 timestamp in RFC3339 format indicating the start of the interval filter (e.g., 2025-01-15T14:00:00Z)",
+							Name: generalFlagStart,
+							Usage: "ISO-8601 timestamp in RFC3339 format indicating the start of the interval filter (e.g., 2025-01-15T14:00:00Z). " +
+								"Defaults to 1 day ago if not specified",
```

Comment:
(minor) instead of including this `Defaults to` in the usage text, can we use the `DefaultText` field?